src/common/gui/main-styles.ts

```diff
@@ -2311,9 +2311,24 @@ styles.registerStyle("main", () => {
 			padding: "35%",
 			margin: "-35% -35%",
 		},
-		".color-picker": {
-			height: px(30),
-			width: px(100),
+		".custom-color-container .text-field": {
```

Comment:
with these few we are kind of reaching into the `TextField` and might break eventually but also the `style` attr is said to be "temporary" and "do not use" so :shrug: we can keep it I guess